tools/GraphClust/CollectResults/evaluation.py

```diff
@@ -46,3 +47,31 @@ def sh(script):
     toWrite += listOfClasses[i] + "\t" + listOfClusters[i] + '\n'
 with open("RESULTS/fullTab.tabular", "w") as full:
     full.write(toWrite)
+
+
+listOfClasses = []
+listOfClusters = []
+pattern = re.compile("^RF.*$")
+
+with open("RESULTS/fullTab.tabular", 'r') as f:
```

Comment:
@eteriSokhoyan It seems one can check the pattern on first element of listOfClasses and not open the file second time. Do yo agree?